docs/src/pages/components/tooltips/CustomizedTooltips.tsx

```diff
@@ -109,6 +109,10 @@ function ArrowTooltip(props: TooltipProps) {
   );
 }
 
+ArrowTooltip.propTypes = {
+  title: PropTypes.node,
+};
```

Comment:
as any?

Reply:
We only "hide" them from the type checker if our propTypes are not as strict as the actual type definitions. Using propTypes with TypeScript requires an exact match e.g. ```ts
interface Props {
  options: { checkMe: number }
}

function Component(props: Props) {}

// rejected
Component.propTypes = { 
	options: PropTypes.object.isRequired,
}

// passes
Component.propTypes = { 
	options: PropTypes.shape({ checkMe: PropTypes.number.isRequired }).isRequired,
}
```

In that case they match so no need for `any`.